src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs

```diff
@@ -206,12 +207,14 @@ public async Task<OrchestratorResponse<ApprenticeshipDetailsViewModel>> GetAppre
 
                 apprenticeship.HashedAccountId = hashedAccountId;
 
+                var includeFrameworks = commitmentData.Commitment.ApprenticeshipEmployerTypeOnApproval != ApprenticeshipEmployerType.NonLevy;
+
                 return new OrchestratorResponse<ExtendedApprenticeshipViewModel>
                 {
                     Data = new ExtendedApprenticeshipViewModel
                     {
                         Apprenticeship = apprenticeship,
-                        ApprenticeshipProgrammes = await GetTrainingProgrammes(true)
+                        ApprenticeshipProgrammes = await GetTrainingProgrammes(includeFrameworks)
```

Comment:
Are there any tests needing changing to cover this logic change

Reply:
No coverage here :-(